packages/contentful/src/crn/queries/research-outputs.queries.ts

```diff
@@ -195,6 +195,20 @@ export const researchOutputContentQueryFragment = gql`
         endDate
       }
     }
+    versions {
+      versionsCollection {
```

Comment:
although this comment could go anywhere else just as well i am going to leave it here - i found it hard to understand your implementation for the versions, and I don't think this aligns with what we discussed. 

So if I understand correctly you would like to have research output point at the output versions element to then point back to RO? so basically each version would just be a different research output? See, this would make things a lot more complicated later.

What we agreed to, at least to what I can remember, is that the versions content model would STORE the title, link and would not point back at RO. So this would just be a way to track the changes of these fields without an actual version history.